keps/sig-node/3008-cri-class-based-resources/README.md

```diff
@@ -1028,6 +1069,11 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+Implementation phase 1: No.
```

Comment:
Surely there will be a unit test of some sort for the feature gate? Is there some test we can do to show that it can be disabled?

In other words, nobody should ever answer "No" to this question - it's more like a checkpoint than a real question :)

Reply:
Yes, this was a slight omission on my part 🙄 I now added a mention about kubelet unit tests.